src/camera/v4l2/SDL_camera_v4l2.c

```diff
@@ -20,46 +20,41 @@
 */
 #include "SDL_internal.h"
 
-#ifdef SDL_VIDEO_CAPTURE
-
-#include "SDL3/SDL.h"
-#include "SDL3/SDL_video_capture.h"
-#include "SDL_sysvideocapture.h"
-#include "SDL_video_capture_c.h"
-#include "SDL_pixels_c.h"
-#include "../thread/SDL_systhread.h"
+#ifdef SDL_CAMERA_V4L2
+
+#include "../SDL_syscamera.h"
+#include "../SDL_camera_c.h"
+#include "../../video/SDL_pixels_c.h"
+#include "../../thread/SDL_systhread.h"
 #include "../../core/linux/SDL_evdev_capabilities.h"
 #include "../../core/linux/SDL_udev.h"
 #include <limits.h>      /* INT_MAX */
 
-#define DEBUG_VIDEO_CAPTURE_CAPTURE 1
-
-#if defined(__linux__) && !defined(__ANDROID__)
+#define DEBUG_CAMERA 1
```

Comment:
Do we keep this enabled by default?
This causes `INFO: Added video capture ID: 1 /dev/video0 (usb-0000:00:14.0-5) (total: 1)` to be printed for (almost) all SDL tests.

Reply:
No, we should not.